scripts/pre_commit_linter.py

```diff
@@ -717,9 +726,12 @@ def _check_comments(all_files):
 
                 if line.startswith('#') and not next_line.startswith('#'):
                     # Check that the comment ends with the proper punctuation.
-                    if (line[-1] not in
-                            ALLOWED_TERMINATING_PUNCTUATIONS) and (
-                                not any(word in line for word in EXCLUDED_PHRASES)):
+                    last_char_is_invalid = line[-1] not in (
+                        ALLOWED_TERMINATING_PUNCTUATIONS)
+                    some_word_is_present_in_excluded_phrases = not any(
```

Comment:
Actually, looking at this again -- shouldn't it be no_word_is_...?

Reply:
Ditto below.

Reply:
Yes, right! Making necessary changes.

Reply:
Done. Thanks!